galgebra/ga.py

```diff
@@ -815,28 +820,37 @@ def _build_bases(self):
             self.blade_super_scripts.append(super_scripts)
 
         if self.debug:
-            printer.oprint('indexes', self.indexes, 'list(indexes)', self.indexes_lst,
-                            'blades', self.blades, 'list(blades)', self.blades_lst,
+            printer.oprint('indexes', self.indexes, 'list(indexes)', self._all_indexes_lst,
+                            'blades', self.blades, 'list(blades)', self._all_blades_lst,
                             'blades_to_indexes_dict', self.blades_to_indexes_dict,
                             'indexes_to_blades_dict', self.indexes_to_blades_dict,
                             'blades_to_grades_dict', self.blades_to_grades_dict,
                             'blade_super_scripts', self.blade_super_scripts)
             if not self.is_ortho:
-                printer.oprint('bases', self.bases, 'list(bases)', self.bases_lst,
+                printer.oprint('bases', self.bases, 'list(bases)', self._all_bases_lst,
                                 'bases_to_indexes_dict', self.bases_to_indexes_dict,
                                 'indexes_to_bases_dict', self.indexes_to_bases_dict,
                                 'bases_to_grades_dict', self.bases_to_grades_dict)
 
         # create the Mv wrappers
-        self.mv_blades_lst = [
+        self._all_mv_blades_lst = [
             mv.Mv(obj, ga=self)
-            for obj in self.blades_lst
+            for obj in self._all_blades_lst
         ]
         self.mv_basis = [
             mv.Mv(obj, ga=self)
             for obj in self.basis
         ]
 
+        # TODO[gh-64]: For compatibility with old behavior, the public
+        # properties do not include the scalar. We should consider making the
+        # breaking change such that they do.
+        self.indexes_lst = self._all_indexes_lst[1:]
+        self.blades_lst = self._all_blades_lst[1:]
+        self.mv_blades_lst = self._all_mv_blades_lst[1:]
+        if not self.is_ortho:
+            self.bases_lst = self._all_bases_lst[1:]
```

Comment:
These lines cause:

![image](https://user-images.githubusercontent.com/425260/69900422-1d2ae400-136b-11ea-93e0-8f0dd0e74406.png)

Again, I don't think it's really viable to try and fix that as part of this patch. Fixing #64 would remove all these lines again.